backend/gsr_booking/api_wrapper.py

```diff
@@ -532,38 +311,175 @@ def get_affiliation(self, email):
         else:
             return "Other"
 
-    def cancel_room(self, user, booking_id):
-        """Cancels room"""
-        gsr_booking = get_object_or_404(GSRBooking, booking_id=booking_id)
-        if gsr_booking:
-            if user != gsr_booking.user and user != gsr_booking.reservation.creator:
+            
+class BookingHandler:
+    def __init__(self, WBW=None, LBW=None):
+        self.WBW = WBW or WhartonBookingWrapper()
```

Comment:
Oh interesting does this work? Or would we need to do `WBW if WBW else WhartonBookingWrapper()`?

Reply:
this works. Python momes haha